modules/search-service-impl/src/main/java/org/opencastproject/search/impl/persistence/SearchServiceDatabaseImpl.java

```diff
@@ -480,4 +521,21 @@ private SearchEntity getSearchEntity(String id, EntityManager em) {
       return null;
     }
   }
+  /**
+   * Gets a search entity by it's id, using the current organizational context.
+   *
+   * @param id
+   *          the series identifier
+   * @param em
+   *          an open entity manager
+   * @return the search entity, or null if not found
+   */
+  private SearchEntity getSeriesSearchEntity(String id, EntityManager em) {
+    Query q = em.createNamedQuery("Search.findBySeriesId").setParameter("seriesId", id);
+    try {
+      return (SearchEntity) q.getSingleResult();
```

Comment:
This will return a single result. The database table stores events. A series can have multiple events.Just retrieving – and later marking as deleted – the first event of a series is a bit weird.

Reply:
A Series has its own id. It will not return multiple events here if you search for an id from a series with multiple events.
then the quuery should serch for the field is part of. 
I hope it is implemented like this if not.. urgh

Reply:
So I assume this change is actually no longer necessary?